From 61c78979eab39ead47a91aeacad43ad72d162052 Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@kernel.org>
Date: Wed, 8 May 2019 21:58:44 +0300
Subject: Makefile.am: Fix a race condition where ell/ell.h was written before
 ell/ existed

Signed-off-by: Adrian Bunk <bunk@kernel.org>
Upstream-Status: Pending

---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index a569c4a3..d2e310d5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1107,6 +1107,7 @@ ell/internal: Makefile
 	done > $@
 
 ell/ell.h: Makefile
+	$(AM_V_at)$(MKDIR_P) ell
 	$(AM_V_at)echo -n > $@
 	$(AM_V_GEN)for f in $(ell_headers) ; do \
 		echo "#include <$$f>" >> $@ ; \
-- 
2.20.1

